compiler/luci/export/src/CircleTensorExporter.cpp

```diff
@@ -132,6 +132,7 @@ class MultiOutputDetector final : public luci::CircleNodeMutableVisitor<bool>
   {
     auto outs = loco::succs(node);
     assert(outs.size() == count);
+    (void)count; // for unused variable error in release build
```

Comment:
(Optional) How about change above assert as runtime_error?

Reply:
Behind this is that import will check the condition and throw if not. This assert is just to be sure the condition is OK.